### PR TITLE
Fix Selector to use heapster, influxdb and grafana on Minikube

### PR DESCRIPTION
**Pull request by tdemaret**
Fix with correct label and selector

### CODE REVIEW

**Comment by dlorenc** [comment]
@minikube-bot ok to test

File: deploy/addons/heapster/grafana-svc.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```
Comment (dlorenc) [inline deploy/addons/heapster/grafana-svc.yaml]: Could you switch this to Reconcile?
    addonmanager.kubernetes.io/mode: Reconcile

File: deploy/addons/heapster/heapster-rc.yaml [hunk 2]
```diff
@@ -16,7 +15,7 @@ spec:
   template:
     metadata:
       labels:
-        kubernetes.io/cluster-service: 'true'
+        addonmanager.kubernetes.io/mode: Reconcile
```
Comment (dlorenc) [inline deploy/addons/heapster/heapster-rc.yaml]: I think this actually needs to go up on the RC labels, from looking here:
https://github.com/kubernetes/kubernetes/blob/b6b3ff59be40be0f907e75d4eebbae4246dbe81f/cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

File: deploy/addons/heapster/heapster-svc.yaml [hunk 3]
```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```
Comment (dlorenc) [inline deploy/addons/heapster/heapster-svc.yaml]: Reconcile.

File: deploy/addons/heapster/influxGrafana-rc.yaml [hunk 4]
```diff
@@ -26,11 +25,12 @@ spec:
   selector:
     addonmanager.kubernetes.io/mode: Reconcile
     name: influxGrafana
+    #name: influxdb-grafana
```
Comment (dlorenc) [inline deploy/addons/heapster/influxGrafana-rc.yaml]: Remove comment?

File: deploy/addons/heapster/influxGrafana-rc.yaml [hunk 5]
```diff
@@ -26,11 +25,12 @@ spec:
...
   template:
     metadata:
       labels:
-        kubernetes.io/cluster-service: 'true'
         name: influxGrafana
+        addonmanager.kubernetes.io/mode: Reconcile
```
Comment (dlorenc) [inline deploy/addons/heapster/influxGrafana-rc.yaml]: I think this needs to go up on the RC.

File: deploy/addons/heapster/influxdb-svc.yaml [hunk 6]
```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
     kubernetes.io/name: monitoring-influxdb
```
Comment (dlorenc) [inline deploy/addons/heapster/influxdb-svc.yaml]: Reconcile.

File: deploy/addons/heapster/heapster-rc.yaml [hunk 7]
```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: ReplicationController
 metadata:
   labels:
```
Comment (dlorenc) [inline deploy/addons/heapster/heapster-rc.yaml]: I think we also need a Reconcile in here.

Comment (tdemaret) [inline deploy/addons/heapster/heapster-rc.yaml]: Sorry, it's fixed

**Comment by tdemaret** [comment]
All is correct ?

**Comment by andyp1per** [comment]
Any chance this could be fixed? I believe it prevents hpa from working.

**Comment by andyp1per** [comment]
I can confirm that with these changes hpa (and the hpa samples) are working

**Pull request merged**